docs/get-started.md

```diff
@@ -157,27 +157,25 @@ Launch the Unity editor. It should automatically detect the project but if it do
 #### Before you start, apply these quick bugfixes:
 <%(#Expandable title="Fix Shaders")%>
 
-There is a bug in the current preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.
+There is a bug in the preview version of the [High Definition Render Pipeline](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended. We've [reported](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/) the issue to Unity.
```

Comment:
Is this necessary info for a user at this point?

Reply:
guess it helps, if people want to track the progress of a bug. though I would expect that in the known issues page and not here. Having a link to the unity forums in the get started guide might get them sidetracked

Reply:
seeing as we have it also in the known issues page, would definitely say that we should remove that sentence here